LICENSE

```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2021, iRODS
+Copyright (c) 2021, The University of North Carolina at Chapel Hill
```

Comment:
can we make this its own PR?   instead of with the -260k LOC?

Reply:
Removed.